include/sir/config.h

```diff
@@ -69,10 +69,33 @@
 # endif
 
 /**
- * Define to use <CR><LF> line endings, otherwise use <LF>.
+ * Define the system carriage return (CR) character.
  */
-# if !defined(SIR_EOL_CRLF)
-#  undef SIR_EOL_CRLF
+# if !defined(SIR_EOL_CR)
+#  define SIR_EOL_CR "\r"
+# endif
+
+/**
+ * Define the system line feed (LF) character.
+ */
+# if !defined(SIR_EOL_LF)
+#  define SIR_EOL_LF "\n"
+# endif
+
+/**
+ * Define to have libsir use CRLF line endings, otherwise use LF.
+ */
+# if !defined(SIR_USE_EOL_CRLF)
+#  undef SIR_USE_EOL_CRLF
+# endif
+
+/**
+ * Define the end of line sequence.
```

Comment:
`The end of line sequence. If SIR_USE_EOL_CRLF is defined, the sequence will be SIR_EOL_CR + SIR_EOL_LF; otherwise just SIR_EOL_LF.`